harness/src/main/java/org/jboss/pvt/harness/validators/VersionFormatCheckValidator.java

```diff
@@ -0,0 +1,81 @@
+package org.jboss.pvt.harness.validators;
+
+import org.jboss.pvt.harness.configuration.PVTConfiguration;
+import org.jboss.pvt.harness.exception.PVTException;
+import org.jboss.pvt.harness.utils.DirUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.File;
+import java.io.FileFilter;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.List;
+import java.util.regex.Pattern;
+
+/**
+ * @author <a href="mailto:yyang@redhat.com">Yong Yang</a>
+ */
+public class VersionFormatCheckValidator implements Validator {
+
+    private final Logger logger = LoggerFactory.getLogger( getClass() );
+
+    private final String DEFAULT_FORMAT = "-redhat-\\d+";
+
+    @Override
+    public boolean validate(PVTConfiguration pvtConfiguration) throws PVTException {
+        File eapDir = pvtConfiguration.getDistributionDirectory();
+
+        Collection<File> jarFiles = DirUtils.listFilesRecursively( eapDir, new FileFilter() {
+            public boolean accept(File pathname) {
+                return  pathname.isFile() && pathname.getName().endsWith(".jar");
+            }
+        });
+
+        logger.info("Jars: " + Arrays.toString(jarFiles.toArray()));
+
+
+        List<File> failedJars = new ArrayList<>();
+
+        Pattern pattern = Pattern.compile(getVersionFormat(pvtConfiguration));
+
+        for (File jarFile : jarFiles) {
+            if (jarFile.isFile()) {
+
+                if (isFilter(pvtConfiguration, jarFile)) {
+                    logger.warn("Excluding Jar, ", jarFile);
+                    continue;
+                }
+
+                if(!pattern.matcher(jarFile.getName()).matches()) {
+                    failedJars.add(jarFile);
```

Comment:
Thanks very much for pointing the PME code. 
Yes, according to https://docs.engineering.redhat.com/display/JP/Productization+Generic+Test+Case, we do need the OSGi version format checking.
@yongyang It will be ideal to have the implementation. 
We might not be able to complete in one PR but would be ideal to have it